docs/source/releasenotes/0.13.1.rst

```diff
@@ -0,0 +1,179 @@
+====================
+What's New in 0.13.1
+====================
+
+.. currentmodule:: openmc
+
+-------
+Summary
+-------
+
+This release of OpenMC includes many bug fixes as well as improvements in
+geometry modeling, mesh functionality, source specification, depletion
+capabilities, and other general enhancements. The depletion module features a
+new transport operator, :class:`openmc.deplete.IndependentOperator`, that allows
+a depletion calculation to be performed using arbitrary one-group cross sections
+(e.g., generated by an external solver). The track file generation capability
+has been significantly overhauled and a new :class:`openmc.Tracks` class was
+introduced to allow access to information in track files from the Python API.
+Support has been added for new ENDF thermal scattering evaluations that use
+mixed coherent/incoherent elastic scattering.
+
+------------------------------------
+Compatibility Notes and Deprecations
+------------------------------------
+
+- The ``openmc.deplete.Operator`` class has been renamed
+  :class:`openmc.deplete.CoupledOperator`.
+- The ``openmc.deplete.ResultsList`` class has been renamed to
+  :class:`openmc.deplete.Results` and no longer requires you to call the
+  ``from_hdf5()`` method in order to create it; instead, you can directly
+  instantiate it.
+- A few methods that represent k-effective have been renamed for the sake of
+  consistency:
+
+  - ``openmc.StatePoint.k_combined`` is now :attr:`openmc.StatePoint.keff`
+  - ``openmc.deplete.ResultsList.get_eigenvalue`` is now
+    :meth:`openmc.deplete.Results.get_keff`
+
+- The :class:`openmc.stats.SphericalIndependent` class, which used to
+  accept a distribution for ``theta`` now accepts a distribution for ``cos_theta``
+  instead in order to more easily handle the common case of specifying a uniform
+  spatial distribution over a sphere (also see the new
+  :func:`openmc.stats.spherical_uniform` function).
+
+- If you are building OpenMC from source, note that several of our CMake options
+  have been changed:
+
+  ==========  ======================
+  Old option  New option
+  ==========  ======================
+  debug       ---
+  optimize    ---
+  profile     OPENMC_ENABLE_PROFILE
+  coverage    OPENMC_ENABLE_COVERAGE
+  openmp      OPENMC_USE_OPENMP
+  ---         OPENMC_USE_MPI
+  dagmc       OPENMC_USE_DAGMC
+  libmesh     OPENMC_USE_LIBMESH
+  ==========  ======================
+
+  The ``debug`` and ``optimize`` options have been removed; instead, use the
+  standard `CMAKE_BUILD_TYPE
+  <https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html>`_
+  variable.
+
+------------
+New Features
+------------
+
+- An :class:`openmc.model.IsogonalOctagon` composite surface.
```

Comment:
```suggestion
- An :class:`openmc.model.IsogonalOctagon` composite surface.
- An :class:`openmc.model.CylinderSector` composite surface.
```

Reply:
I did incorporate this but combined them into a single bullet point